### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by vladkasianenko** [stylelint/stylelint#7296]
### What minimal example or steps are needed to reproduce the bug?

![image](https://github.com/stylelint/stylelint/assets/14053728/8ed86820-7b55-4105-a349-4c04ea59a9af)

### What minimal configuration is needed to reproduce the bug?

![image](https://github.com/stylelint/stylelint/assets/14053728/c4a7684d-d5b5-4c0f-8e7d-783718871cc2)

### How did you run Stylelint?

Automatically in VS Code

### Which Stylelint-related dependencies are you using?

"stylelint": "^15.11.0",
    "stylelint-config-standard": "^34.0.0",
    "stylelint-config-standard-scss": "^11.0.0",

### What did you expect to happen?

Not to give me an error.

### What actually happened?

An error.

**Comment by ybiquitous** [comment]
@vladkasianenko From the screenshots, you may try linting a JS file instead of a SCSS file, right? Perhaps, you may need to specify a custom syntax for your JS files. Please see https://github.com/stylelint/awesome-stylelint#custom-syntaxes

Also, we'd appreciate it if you could provide a minimal reproducible [demo](https://stylelint.io/demo/) or repository.

**Comment by jeddy3** [comment]
Closing as related to the VSCode extension rather than Stylelint itself. Feel free to reopen with a repo demo if that's not the case, though.

@vladkasianenko If you've configured Stylelint to only lint SCSS, you need to then configure the VS Code to only lint SCSS too using the [`.validate`](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint#stylelint.validate) option:

```
{
  "stylelint.validate": ["scss"]
}
```
